packages/account-sdk/src/interface/payment/pay.ts

```diff
@@ -103,12 +102,7 @@ export async function pay(options: PaymentOptions): Promise<PaymentResult> {
       logPaymentError({ amount, testnet, correlationId, errorMessage });
     }
 
-    // Return error result
-    return {
-      success: false,
-      error: errorMessage,
-      amount: amount,
-      to: to as Address,
-    };
+    // Re-throw the original error
```

Comment:
the comment on line 29 needs to be updated too to reflect the new behavior.

Reply:
done-zo